script/rummager_export.rb

```diff
@@ -57,9 +57,20 @@ def export_classes(classes_to_index, id_groups, &block)
 end
 
 def output_es_line(obj, output)
-  s = obj.search_index
-  output.puts %Q[{"index": {"_type": "edition", "_id": "#{s['link']}"}}]
-  output.puts s.to_json
+  max_retry_count = 5
+  begin
+    search_index = obj.search_index
+  rescue
+    max_retry_count -= 1
```

Comment:
It would be really good to log a warning message here, so we know if it's beginning to fail.  Also, I'd suggest sleeping for a little bit here (eg, 5 seconds) to give an overloaded downstream service a chance to recover.